.github/workflows/tests.yml

```diff
@@ -35,8 +31,67 @@ jobs:
       - name: "Run C/C++ linter"
         run: ./bin/run_clang_tidy.sh
 
+  cpp-funcs:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-20.04
+    container:
+      image: faasm/cpp-sysroot:0.0.22
+    defaults:
+      run:
+        working-directory: /__w/faasm/faasm/clients/cpp
```

Comment:
Github actions `checkout` action is much quicker than our branch approach and a bit more readable, but it checks it out to `/__w/<org>/<repo>/`